app/models/game.rb

```diff
@@ -36,4 +36,27 @@ def other_player(player)
   def pass_turn(player)
     update_attributes(active_player_id: other_player(player))
   end
+
+  def stalemate?
+    king_not_in_check? && available_move?
+  end
+
+  def king_not_in_check?
+    king = pieces.find_by(type: 'King', color: current_color)
+    return true unless king.in_check?
+    false
```

Comment:
Oh no sorry I didn't mean you didn't need the method, I think the `king_not_in_check` method is great, I was just talking about the last part of it lines 46 & 47. Instead of using those two lines you could just put `!king.in_check?`. So definitely keep the method and if you want you can change those last two lines, but like I said what you have works fine as well so not totally necessary. :smile: